controller/features_test.go

```diff
@@ -25,3 +33,9 @@ func TestFeatures(t *testing.T) {
 	//	test.ListFeaturesNotFound(t, context.Background(), svc, ctrl)
 	//})
 }
+func createValidContext() context.Context {
```

Comment:
so you should have an `invalid` context, too, in order to verify that you get no enabled feature for a non-Red Hat user